src/lib/logical_query_plan/import_node.cpp

```diff
@@ -2,46 +2,60 @@
 
 #include <cstddef>
 #include <memory>
-#include <sstream>
+#include <optional>
 #include <string>
 
+#include <boost/algorithm/string.hpp>
 #include <boost/container_hash/hash.hpp>
 
+#include "magic_enum.hpp"
+
 #include "import_export/file_type.hpp"
 #include "logical_query_plan/abstract_lqp_node.hpp"
 #include "logical_query_plan/abstract_non_query_node.hpp"
+#include "storage/encoding_type.hpp"
 
 namespace hyrise {
 
 ImportNode::ImportNode(const std::string& init_table_name, const std::string& init_file_name,
-                       const FileType init_file_type)
+                       const FileType init_file_type, const std::optional<EncodingType> init_table_encoding)
     : AbstractNonQueryNode(LQPNodeType::Import),
       table_name(init_table_name),
       file_name(init_file_name),
-      file_type(init_file_type) {}
+      file_type(init_file_type),
+      table_encoding(init_table_encoding) {}
 
 std::string ImportNode::description(const DescriptionMode /*mode*/) const {
-  std::ostringstream stream;
-  stream << "[Import] Name: '" << table_name << "'";
-  return stream.str();
+  auto file_type_str = std::string{magic_enum::enum_name(file_type)};
+  boost::algorithm::to_lower(file_type_str);
+
+  if (table_encoding) {
+    auto file_encoding_str = std::string{magic_enum::enum_name(*table_encoding)};
+    boost::algorithm::to_lower(file_encoding_str);
+    return "[Import] Name: '" + table_name + "' from '" + file_name + "' (" + file_type_str + ") using encoding (" +
+           file_encoding_str + ")";
+  }
+
+  return "[Import] Name: '" + table_name + "' from '" + file_name + "' (" + file_type_str + ")";
```

Comment:
I'm not sure if we require all of these details (they will be printed in the visualization as well and can arbitrarily bloat them). I guess that could be a reason not to include the path that will just be create a mess:
![image](https://github.com/user-attachments/assets/17fb4e91-2417-4859-bc08-45fc3c686c19)

I guess I would prefer a message such as:
```
[Import] 'foo' (file type Auto, encoding Auto) 
[Import] 'foo' (file type Tbl, encoding Unencoded) 
```
(also without forced lowercase). The descriptions won't be printed on a hot path, so I'd prefer a single return. If it makes things more easy to read, there's also nothing to say against a stream to collect the message.